readthedocs/api/mixins.py

```diff
@@ -30,7 +32,7 @@ class CDNCacheTagsMixin:
 
     def dispatch(self, request, *args, **kwargs):
         response = super().dispatch(request, *args, **kwargs)
-        cache_tags = self._get_cache_tags()
+        cache_tags = getattr(self, "_cache_tags", self._get_cache_tags())
```

Comment:
I also don't think this is needed? I think just adding the mixin to the view will cause them to get set.